docs/variants/asus_kgpe_d16/dasharo-compatibility/cbmem_boot_time_measurment.md

```diff
@@ -0,0 +1,61 @@
+# Booting time measurment with cbmem on Ubuntu OS
+
+## Coldboot time measurment
+
+1. Disconnect DUT from power source.
+
+2. Connect back DUT to power source.
+
+3. Power ON DUT.
+
+4. Get `cbmem`.
```

Comment:
If we write about any external software/scripts, we should add information, from where this one might be downloaded